constants.js

```diff
@@ -240,6 +246,8 @@ const BID_SUGGESTION_RANGE_ENUM = [
     ALL_BID_SUGGESTION,
 ];
 
+const NEUROWEB_BLOCKCHAIN_PREFIX = 'otp';
```

Comment:
If it does not take too much time perhaps we can rename it to 'neuroweb' and adjust the logic elsewhere